src/components/organisms/dashboard/projectsTable.tsx

```diff
@@ -211,38 +212,78 @@ export const DashboardProjectsTable = () => {
 									<Td
 										className="hidden w-1/6 sm:flex"
 										onClick={() => navigate(`/${SidebarHrefMenu.projects}/${id}`)}
-										title={`${totalDeployments} ${t("table.columns.deployments")}`}
+										title={`${totalDeployments} ${t("table.columns.totalDeployments")}`}
 									>
 										<div className="w-full pr-6 text-center">{totalDeployments}</div>
 									</Td>
 									<Td
-										className="-ml-1 hidden w-2/6 pr-2 sm:flex"
+										className="-ml-1 w-2/6 pr-2 sm:flex"
 										onClick={() => navigate(`/${SidebarHrefMenu.projects}/${id}`)}
 									>
 										<div
 											aria-label={t("table.sessionTypes.running")}
 											className={countStyle(SessionStateType.running)}
+											onClick={(event) => {
+												event.stopPropagation();
+												navigate(
+													`/${SidebarHrefMenu.projects}/${id}/deployments/dep_01jg9h2r97e7187j4f8j8fraea/sessions`,
+													{ state: { sessionState: SessionStateType.running } }
+												);
+											}}
+											onKeyDown={() => {}}
```

Comment:
Please:
1. take `onClick` to a separate function
2. attach the same function to the `onKeyDown`

Reply:
changed